doc/sphinx/proof-engine/tactics.rst

```diff
@@ -227,9 +244,8 @@ For a goal :g:`A \/ B`, use :tacn:`left` to replace the current goal with :g:`A`
 
 * :n:`[ {+| {* @intropattern } } ]` — splits an inductive type that has
   :ref:`multiple constructors <intropattern_cons_note>`
-  such as :n:`A \/ B`
-  into multiple subgoals.  The number of :token:`intropattern`\s must be the same as the number of
-  constructors for the matched part.
+  such as :n:`A \/ B` into multiple subgoals.  The number of :token:`intropattern`\s
```

Comment:
What does `\s` do here?

Reply:
The `\` is needed to add an "s" to intropattern without a space but outside the token style (I guess another solution would be to use :token:`intropatterns <intropattern>`, but we've never done this so far).

Reply:
I prefer using `\` so the "s" is in a different font from the token name.  (We probably have nonterminals that only differ by a final "s".)